jaeger-core/src/main/java/com/uber/jaeger/Configuration.java

```diff
@@ -508,13 +591,16 @@ public Integer getFlushIntervalMs() {
     public Integer getMaxQueueSize() {
       return maxQueueSize;
     }
+
+    public SenderConfiguration getSenderConfiguration() {
+      return senderConfiguration;
+    }
   }
 
   /**
    * Holds the configuration related to the sender. A sender can be a {@link HttpSender} or {@link UdpSender}
    *
    */
-  @Getter
```

Comment:
looks like a breaking change

Reply:
Added back, although I think this was added only for tests by @jpkrohling. I don't think we had a requirement of exposing it.